tests/test_enterprise/api/test_views.py

```diff
@@ -2385,6 +2385,77 @@ def test_post_licensed_course_enrollments_license_revoke(
         assert enterprise_course_enrollment.saved_for_later == is_revoked
         assert licensed_course_enrollment.is_revoked == is_revoked
 
+    @ddt.data(
+        # API should return enterprise customer details
+        (False, True, True),
+        # API should not return enterprise customer details
+        (False, False, False),
+        # for staff, API should return enterprise customer details irrespective of linked status
+        (True, True, True),
+        # for staff, API should return enterprise customer details irrespective of linked status
+        (True, False, True),
```

Comment:
Nit: It's more verbose, but I think ddt params as dictionaries (like here: https://github.com/edx/license-manager/blob/a27017cae0b357936b8a22f46802e7a0877008fe/license_manager/apps/api/v1/tests/test_views.py#L952-L990) makes it easier to understand what each test is checking